libraries/AP_BattMonitor/AP_BattMonitor_Logging.cpp

```diff
@@ -16,6 +16,9 @@ void AP_BattMonitor_Backend::Log_Write_BAT(const uint8_t instance, const uint64_
         temperature_cd = temperature * 100.0;
     }
 
+    uint8_t soh_pct = -1;
```

Comment:
Why not use 0 for unknown inline with the MAVLink spec?

Reply:
Txs for the review, changed to 0.